cmd/osbuild-worker/main.go

```diff
@@ -154,6 +157,94 @@ func RunJob(job *worker.Job, store string, uploadFunc func(uuid.UUID, string, io
 				azureMaxUploadGoroutines,
 			)
 
+			if err != nil {
+				r = append(r, err)
+				continue
+			}
+		case *target.KojiTargetOptions:
+			credentials, err := koji.GSSAPICredentialsFromEnv()
+			if err != nil {
+				r = append(r, err)
+				continue
+			}
+
+			// Koji for some reason needs TLS renegotiation enabled.
+			// Clone the default http transport and enable renegotiation.
+			transport := http.DefaultTransport.(*http.Transport).Clone()
+			transport.TLSClientConfig = &tls.Config{
+				Renegotiation: tls.RenegotiateOnceAsClient,
+			}
+
+			k, err := koji.NewFromGSSAPI(options.Server, credentials, transport)
+			if err != nil {
+				r = append(r, err)
+				continue
+			}
+
+			defer func() {
+				err := k.Logout()
+				if err != nil {
+					log.Printf("koji logout failed: %v", err)
+				}
+			}()
+
+			f, err := os.Open(path.Join(outputDirectory, options.Filename))
+			if err != nil {
+				r = append(r, err)
+				continue
+			}
+
+			hash, filesize, err := k.Upload(f, options.UploadDirectory, options.Filename)
+			if err != nil {
+				r = append(r, err)
+				continue
+			}
+
+			build := koji.Build{
+				Name:      job.Id.String(),
+				Version:   "1",
+				Release:   "1",
+				StartTime: time.Now().Unix(),
+				EndTime:   time.Now().Unix(),
+			}
+			buildRoots := []koji.BuildRoot{
+				{
+					ID: 1,
+					Host: koji.Host{
+						Os:   "RHEL8",
+						Arch: "noarch",
```

Comment:
Does this tell koji information about the host the image was built on, or about the image itself?